02_configure_host.sh

```diff
@@ -128,9 +128,17 @@ if [[ $(uname -m) == "aarch64" ]]; then
   echo "libvirt_cdrombus: scsi" >> vm_setup_vars.yml
 fi
 
+ironic_prefix_env_var="ironic_prefix=${CLUSTER_NAME}_"
+# If this is a Two Node OpenShift with Fencing (TNF) installation
+# the name of the entry in the infra_nodes.json must match the hostname
+# of the created host so that the fencing secret can be found
+if  [[ -z "${ENABLE_ARBITER:-}" ]] && [[ "${NUM_MASTERS}" -eq 2 ]]; then
+    ironic_prefix_env_var="ironic_prefix=''"
+fi
+
 ansible-playbook \
     -e @vm_setup_vars.yml \
-    -e "ironic_prefix=${CLUSTER_NAME}_" \
+    -e "${ironic_prefix_env_var}" \
```

Comment:
This might have problematic implications. Since I've added this line, my clusters don't see to get past the ironic inspection phase. I'm wondering if the `ostest_` is used somewhere else to identify the nodes somehow.

Reply:
I decided to avoid messing with the ironic name, since it's also used in the creation of the network bonds. Instead, I just reference the MASTER_HOSTNAME_FORMAT variable in the creation of the secret, which should align the secret name.